stylish.js

```diff
@@ -80,6 +80,9 @@ export default {
       })
     }
 
-    return `\n${underlinedFilename}\n${table(output)}`
+    if (underlinedFilename) {
+      return `\n${underlinedFilename}\n${table(output)}`
+    }
+    return `\n${table(output)}`
```

Comment:
I felt same when I change added.

Just didn't wanted to bloat with more tests for only that.
Let give it a try. :+1: